### PR TITLE
Use VIP status file for config server (and controller) in hosted

### PR DESCRIPTION
**Pull request by hmusum**
The code for setting up VIP status is already in place in config model (see ContainerModelBuilder.addStatusHandlers()),
but depends on value of hostedVespa() in ModelContext.Properties, so have to
make sure that is set for standalone container as well.
Removing config override for vip status, as setting VIP in or out of rotation when not
using a VIP status file is handled when bootstrapping now.

### CODE REVIEW

File: configserver/src/main/java/com/yahoo/vespa/config/server/ConfigServerBootstrap.java [hunk 1]
```diff
@@ -37,8 +37,10 @@
  * Main component that bootstraps and starts config server threads.
  *
  * If config server has been upgraded to a new version since the last time it was running it will redeploy all
- * applications. If that is done successfully the RPC server will start and the health status code will change from
- * 'initializing' to 'up' and the config server will be put into rotation (start serving status.html with 200 OK)
+ * applications. If that is done successfully the RPC server the health status code will change from
```
Comment (hakonhall) [inline configserver/src/main/java/com/yahoo/vespa/config/server/ConfigServerBootstrap.java]: "RPC server the health status" -> " RPC server will start and the health status" ?

File: configserver/src/test/java/com/yahoo/vespa/config/server/ConfigServerBootstrapTest.java [hunk 2]
```diff
@@ -81,6 +88,33 @@ public void testBootstrap() throws Exception {
...
+        File versionFile = temporaryFolder.newFile();
+        VersionState versionState = new VersionState(versionFile);
+        assertTrue(versionState.isUpgraded());
+
+        RpcServer rpcServer = createRpcServer(configserverConfig);
+        VipStatus vipStatus = createVipStatus(VIP_STATUS_FILE);
+        ConfigServerBootstrap bootstrap = new ConfigServerBootstrap(tester.applicationRepository(), rpcServer,
+                                                                    versionState, createStateMonitor(),
+                                                                    vipStatus, INITIALIZE_ONLY, VIP_STATUS_FILE);
+        assertTrue(vipStatus.isInRotation()); // default is in rotation when using status file
```
Comment (hakonhall) [inline configserver/src/test/java/com/yahoo/vespa/config/server/ConfigServerBootstrapTest.java]: This is for backwards compatibility? Won't this make it impossible for us to use VIP_STATUS_FILE safely?

File: standalone-container/src/main/java/com/yahoo/container/standalone/StandaloneContainerApplication.java [hunk 3]
```diff
@@ -340,4 +345,82 @@ private static void initializeContainerModel(ContainerModel containerModel, Conf
...
+        }
+
+        @Override
+        public URI ztsUrl() {
+            return null;
+        }
+
+        @Override
+        public String athenzDnsSuffix() {
+            return null;
```
Comment (hakonhall) [inline standalone-container/src/main/java/com/yahoo/container/standalone/StandaloneContainerApplication.java]: These should throw UnsupportedOperationException to ensure they're not called, and/or don't cause strange NPE later?

File: configserver/src/main/java/com/yahoo/vespa/config/server/ConfigServerBootstrap.java (hunk 1 above)
Comment (hmusum) [reply configserver/src/main/java/com/yahoo/vespa/config/server/ConfigServerBootstrap.java]: Yes, will fix

File: standalone-container/src/main/java/com/yahoo/container/standalone/StandaloneContainerApplication.java (hunk 3 above)
Comment (hmusum) [reply standalone-container/src/main/java/com/yahoo/container/standalone/StandaloneContainerApplication.java]: Good point

File: configserver/src/test/java/com/yahoo/vespa/config/server/ConfigServerBootstrapTest.java (hunk 2 above)
Comment (hmusum) [reply configserver/src/test/java/com/yahoo/vespa/config/server/ConfigServerBootstrapTest.java]: This is what VipStatus thinks about the status, if a ststus file is used the response is based on whether the file exists or not (see comment for the test).

Comment (hakonhall) [reply configserver/src/test/java/com/yahoo/vespa/config/server/ConfigServerBootstrapTest.java]: Got it, 👍

**Review by hakonhall — APPROVED** [review]

**Comment by hmusum** [comment]
I'll merge this tomorrow morning

**Pull request merged**